### PR TITLE
Balance Bugfix

### PR DESCRIPTION
**Pull request by SidonieBouthors**
Bugfixes for creating or editing a balance item

### CODE REVIEW

File: app/src/main/java/com/github/se/assocify/ui/screens/treasury/accounting/balance/BalancePopUpScreen.kt [hunk 1]
```diff
@@ -167,13 +165,14 @@ fun BalancePopUpScreen(balanceDetailedViewModel: BalanceDetailedViewModel) {
 
             // The amount field
             item {
+              Log.e("BalancePopUpScreen", "receiptUid: $receiptUid")
```
Comment (sarahbadr17) [inline app/src/main/java/com/github/se/assocify/ui/screens/treasury/accounting/balance/BalancePopUpScreen.kt:168]: remove debug statement

File: app/src/main/java/com/github/se/assocify/ui/screens/treasury/accounting/balance/BalancePopUpScreen.kt [hunk 2]
```diff
@@ -316,6 +315,7 @@ fun BalancePopUpScreen(balanceDetailedViewModel: BalanceDetailedViewModel) {
                           nameString, receiptUid, amountString, assignee, descriptionString, date)
 
                       if (balanceModel.creating && amountString.toDoubleOrNull() != null) {
+                        Log.e("BalancePopUpScreen", "Creating balance item")
```
Comment (sarahbadr17) [inline app/src/main/java/com/github/se/assocify/ui/screens/treasury/accounting/balance/BalancePopUpScreen.kt:318]: remove debug statement

File: app/src/main/java/com/github/se/assocify/ui/screens/treasury/accounting/balance/BalancePopUpScreen.kt [hunk 3]
```diff
@@ -330,6 +330,7 @@ fun BalancePopUpScreen(balanceDetailedViewModel: BalanceDetailedViewModel) {
                                 mutableStatus)
                         balanceDetailedViewModel.saveCreation(newBalanceItem)
                       } else if (amountString.toDoubleOrNull() != null) {
+                        Log.e("BalancePopUpScreen", "Editing balance item")
```
Comment (sarahbadr17) [inline app/src/main/java/com/github/se/assocify/ui/screens/treasury/accounting/balance/BalancePopUpScreen.kt:333]: same

**Review by sarahbadr17 — CHANGES_REQUESTED** [review]
Just some comments and debug statements left!

Test it on my phone and it works! Thank you

File: app/src/main/java/com/github/se/assocify/ui/screens/treasury/accounting/balance/BalancePopUpScreen.kt (hunk 1 above)
Comment (SidonieBouthors) [reply app/src/main/java/com/github/se/assocify/ui/screens/treasury/accounting/balance/BalancePopUpScreen.kt:168]: My bad! Good catch :)

**Review by sarahbadr17 — APPROVED** [review]
Looks good to me! You can merge

**Pull request merged**